util/backup/advanced/restore-redis-infisical.sh

```
@@ -0,0 +1,592 @@
+        echo ""
+        show_usage
+        exit 1
+        ;;
+esac
+
+# =============================================================================
+# INFISICAL DELEGATION
+# =============================================================================
+
+export PROJECT_ID="6ae965bb-3cf8-4d44-9658-9cd4d58f754c"
+export BACKUP_ENCRYPTION_KEY=$(infisical secrets get BACKUP_ENCRYPTION_KEY --projectId="$PROJECT_ID" --env="$ENVIRONMENT" --plain)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)